test/integration/targets/win_dns_record/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Clean slate
+  import_tasks: clean.yml
+  vars:
+    fail_on_missing: false
+
+- block:
```

Comment:
While you've added the skip in the aliases, you should still check and skip if necessary when running on older versions of Windows. Usually it's a simple

```
- win_shell: if (Get-Command -Name ... -ErrorAction SilentlyContinue) { $true } else { $false }
  register: run_tests

- name: run tests on supported hosts
  when: run_tests.stdout | trim | bool
  block:
```

Reply:
Alright, I know when I'm beat (@Daniel-Sanchez-Fabregas mentioned something similar).

But keep in mind that the relevant cmdlets aren't available _even on supported platforms_ unless/until the DNS feature is installed.

So I've added the cmdlet tests, but also put OS-version tests in front of them to avoid installing the DNS feature unnecessarily. That has the effect of making the cmdlet test redundant, however.

Reply:
Ah sorry that makes sense, I would just do the OS check then, sorry for the confusion.

Reply:
Done.